helm/manifest.yaml

```diff
@@ -2063,16 +2070,25 @@ metadata:
   name: flyte-grpc
   namespace: flyte
   annotations:
+    alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-2:832573439839:certificate/a7db79d4-4f8c-4c7b-a4f4-109bf2fb5efa
     alb.ingress.kubernetes.io/group.name: flyte
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}]'
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/tags: service_instance=production
     kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/backend-protocol-version: HTTP2
     nginx.ingress.kubernetes.io/backend-protocol: GRPC
 spec:
   rules:
     - http:
         paths:
+          #
```

Comment:
what was this for?

Reply:
This is coming from the revert, but we can regenerate this file if needed in your branch or delete it since its not used directly for the installation.

Reply:
Specifically this is coming from
```
   # {{- if .Values.common.ingress.albSSLRedirect }}
          # - backend:
          #     serviceName: ssl-redirect
          #     servicePort: use-annotation
          #   path: /*
          #   pathType: ImplementationSpecific
          # {{- end }}
```
From templates/common/ingress.yaml as we have now enabled albSSLRedirect  with self signed certs